### ISSUE TITLE
I have such a scenario. How to implement the for loop, traverse the values in the list, and then execute an app

### ISSUE THREAD

**Issue by 219liangyi** [frikky/Shuffle#627]
I have such a scenario. How to implement the for loop, traverse the values in the list, and then execute an app

The current docs documents are used to obtain the value of a key in the list, but my more common scenario is that I get a list and need to traverse the values in the list and execute the next app one by one. Batch operations are not supported
Each key calls the next app once

**Comment by frikky** [comment]
Hey! This is supported, and our documentation is outlining it here: https://shuffler.io/docs/workflows#passing_lists

This will explicitly loop/traverse the list as long as there are more items in it

**Comment by 219liangyi** [comment]
https://shuffler.io/docs/workflows#passing_lists Maybe I made a mistake. Let me rephrase it
After an app obtains a list, for example, the list has 10 pieces of data. I expect to traverse these 10 pieces of data and pass them to the next app to execute the next app. The final result is that the next app is executed 10 times.
The current support is that I get a list, and I extract all the values in it and pass them to the next app. However, more of my scenarios do not support batch transmission of all data at one time, and can only be executed one by one
